src/webhook/group_validation.py

```diff
@@ -31,6 +31,11 @@ def handle_request():
     body_dict = request.json['request']
     group_name = body_dict['object']['metadata']['name']
     userinfo = body_dict['userInfo']
+    if userinfo['username'] == "kube:admin":
```

Comment:
I'm not sure if the user will come as `kube:admin` or just `admin` in the AdmissionReview object. 
Were you able to confirm?

Reply:
The other group providing permissions higher than `osd-sre-admins` is `osd-sre-cluster-admins`, should we also address it here?

Reply:
I can confirm that it's `kube:admin`. Here is a sample request body (forgive the Python format, but it's close enough to JSON):

```python
{
  'kind': 'AdmissionReview', 
  'apiVersion': 'admission.k8s.io/v1beta1', 
  'request':
  {
    'uid': 'd0f53355-0d1b-11ea-8394-0a580a820038', 
    'kind': 
    {
      'group': 'user.openshift.io', 
      'version': 'v1',
      'kind': 'Group'
    },
    'resource': 
    {
      'group': 'user.openshift.io',
      'version': 'v1',
      'resource': 
      'groups'
    }, 
    'operation': 
    'CREATE', 
    'userInfo': 
    {
      'username': 
      'kube:admin', 
      'groups': 
      [
        'system:cluster-admins', 
        'system:authenticated'
      ], 
      'extra': 
      {
        'scopes.authorization.openshift.io': 
        [
          'user:full'
        ]
      }
    }, 
    'object': 
    {
      'kind': 
      'Group', 
      'apiVersion': 
      'user.openshift.io/v1', 
      'metadata': 
      {
        'name': 'osd-sre-foo123', 
        'uid': 'd0f52f1d-0d1b-11ea-8394-0a580a820038', 
        'creationTimestamp': '2019-11-22T11:32:50Z'
      }, 
      'users': None
    }, 
    'oldObject': None, 
    'dryRun': False
  }
}
```